packages/paste-website/src/components/icons-list/IconsList.tsx

```diff
@@ -0,0 +1,249 @@
+              }}
+            />
+          </Box>
+        </Column>
+        <Column span={7}>
+          {iconsList.ui.length > 0 && (
+            <>
+              <Heading as="h2" variant="heading20">
+                UI Icons
+              </Heading>
+              <Paragraph>
+                UI icons are the general collection of icons that can be used in most user interfaces. They are most
+                commonly used in conjunction with performing an interaction. Be sure to follow the{' '}
+                <SiteLink to="/icons/usage-guidelines">usage guidelines</SiteLink> carefully when placing an icon in
+                your UI.
+              </Paragraph>
+              <Composite
+                aria-label="UI Icons"
+                as={Box}
+                marginX="spaceNegative40"
+                marginBottom="space70"
+                display="flex"
+                flexWrap="wrap"
+                role="listbox"
+                {...uiComposite}
+              >
+                {iconsList.ui.map((icon, index) => (
+                  <IconListItem
+                    aria-selected={selectedIcon != null && selectedIcon.name === icon.name}
+                    key={iconKeySeed(`icon-${index}`)}
+                    icon={icon}
+                    setSelectedIcon={setSelectedIcon}
+                    {...uiComposite}
+                  />
+                ))}
+              </Composite>
+            </>
+          )}
+
+          {iconsList.products.length > 0 && (
+            <>
+              <Heading as="h2" variant="heading20">
+                Product Icons
+              </Heading>
+              <Paragraph>
+                Product icons represent the various Twilio products. Ideally you should be pairing these with a visible
+                name of the product. Be sure to follow the{' '}
+                <SiteLink to="/icons/usage-guidelines">usage guidelines</SiteLink> carefully when placing an icon in
+                your UI.
+              </Paragraph>
+              <Composite
+                as={Box}
+                aria-label="Product icons"
+                marginX="spaceNegative40"
+                marginBottom="space70"
+                display="flex"
+                flexWrap="wrap"
+                role="listbox"
+                {...productComposite}
+              >
+                {iconsList.products.map((icon, index) => (
+                  <IconListItem
+                    aria-selected={selectedIcon != null && selectedIcon.name === icon.name}
+                    key={iconKeySeed(`icon-${index}`)}
+                    icon={icon}
+                    setSelectedIcon={setSelectedIcon}
+                    {...productComposite}
+                  />
+                ))}
+              </Composite>
+            </>
+          )}
+
+          {iconsList.logos.length > 0 && (
+            <>
+              <Heading as="h2" variant="heading20">
+                Logos
```

Comment:
I would merge logo and Product icons ideally. Both are pieces of brand information. I would call the new category "Product Logos"

Reply:
@serifluous wanted the logo category, granted it was in the UI logos section previously, but the request was a logo category

Reply:
The Twilio logo usage guidelines are stricter, and the padding we've given to the Twilio logo icon vs. the product icons is different (to make the required clearspace around the Twilio logo easier to calculate), so I think it makes sense to separate them out.

We could rename the headings if they're confusing.